aws/coreos-docker-servers.tf

```diff
@@ -30,3 +33,23 @@ resource "template_file" "etcd_discovery_url" {
     }
   }
 }
+
+resource "aws_security_group" "docker_node" {
```

Comment:
After discussing it in the team: We will block also the access to port 4243 for any host except the API nodes. 

Security groups won't prevent the app container accessing the local docker daemon, as the communication happens in the same host, but this will be tackled in a different story.